index.d.ts

```diff
@@ -25,8 +25,8 @@ declare namespace jest {
 
 interface Serializer {
   test: (value: any) => boolean;
-  print: (value: any, _serializer?: Function) => string;
-  diffOptions: (valueA: any, valueB: any) => DiffOptions;
+  print: (value: any, _serializer?: any) => any;
```

Comment:
I think we can leave the string return here

Reply:
@thymikee Unfortunately not; the `enzyme-to-json` type declaration for `JestSerializer` has `print` returning their `Json` type which is an object.

https://github.com/adriantoine/enzyme-to-json/blob/d1f86ee82ce014bb3d63f521a5d553307948eff0/index.d.ts#L24

Keeping it as `string` give this error:
```
Type 'JestSerializer' is not assignable to type 'Serializer'.
  Types of property 'print' are incompatible.
    Type '(CommonWrapper: any, serializer: JestSerializer) => Json' is not assignable to type '(value: any, _serializer?: any) => string'.
      Type 'Json' is not assignable to type 'string'.
```

Reply:
Oh, wasn't aware. That's fine then